spec/models/service_template_provision_task_spec.rb

```diff
@@ -86,7 +88,7 @@ def service_resource_id(index, scaling_max)
           :class_name       => 'ServiceProvision_Template',
           :instance_name    => 'clone_to_service',
           :automate_message => 'create',
-          :attrs            => {'request' => 'clone_to_service'},
+          :attrs            => {'request' => 'clone_to_service', 'Service::Service' => @service.id},
```

Comment:
@mkanoor I thought the variable name was lower cased.  I would expect this to be `'Service::service' => @service.id`.